src/theme/index.ts

```diff
@@ -56,31 +56,33 @@ const theme = {
   space,
 }
 
+type SCWUITheme = typeof theme & {
+  linkComponent?: unknown
+}
+
 type RecursivePartial<T> = {
   [P in keyof T]?: RecursivePartial<T[P]>
 }
 
 /**
  * Will extend theme with new theme properties
- * @param {typeof theme} baseTheme the theme you want to extend from, by default it is set to light theme
- * @param {RecursivePartial<typeof theme>} extendedTheme the properties of a new theme you want to apply from baseTheme
+ * @param {SCWUITheme} baseTheme the theme you want to extend from, by default it is set to light theme
+ * @param {RecursivePartial<SCWUITheme>} extendedTheme the properties of a new theme you want to apply from baseTheme
  */
 const extendTheme = ({
   baseTheme = theme,
   extendedTheme,
 }: {
-  baseTheme?: typeof theme
-  extendedTheme: RecursivePartial<typeof theme>
-}) => deepmerge(baseTheme, extendedTheme) as typeof theme
+  baseTheme?: SCWUITheme
+  extendedTheme: RecursivePartial<SCWUITheme>
+}) => deepmerge(baseTheme, extendedTheme) as SCWUITheme
+
+const lightTheme: SCWUITheme = theme
```

Comment:
I don't understand why we create this `lightTheme`, can't we export the existing `theme` and cast it as a `SCWUITheme`?

Reply:
I think it is because that way, the exported theme is correctly typed and can be imported without any `theme is of type any` into an outside project

Reply:
Yeah but can't we simply `export theme as SCWUITheme`?

Reply:
Casting a value will force the type, it's better to set the variable type, not override it

Also SCWUITheme is inferred from theme, meaning we can't do `export theme as SCWUITheme` as we would a circular ref :)

Reply:
@QuiiBz I think the idea is also to make it explicit that we export lightTheme as default theme

Reply:
Thanks for the clarification :)